service/src/main/java/bio/terra/workspace/service/resource/controlled/cloud/azure/AzureStorageAccessService.java

```diff
@@ -224,12 +227,19 @@ public AzureSasBundle createAzureStorageContainerSasToken(
       resourceName = storageData.storageContainerResource().getStorageContainerName();
     }
 
+    var sig = token.split("sig=")[1];
```

Comment:
Really wish the azure libs gave us a better object to work with for SAS's than a raw string.